### PR TITLE
Object tracking and Text detection GA samples

### PR DESCRIPTION
**Pull request by nirupa-kumar**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: samples/analyze.js [hunk 1]
```diff
@@ -301,6 +301,299 @@ async function analyzeVideoTranscription(gcsUri) {
   // [END video_speech_transcription_gcs]
 }
 
+async function analyzeTextGCS(gcsUri) {
+  //[START video_detect_text_gcs]
+  // Imports the Google Cloud Video Intelligence library
+  const Video = require('@google-cloud/video-intelligence').v1;
```
Comment (JustinBeckwith) [inline samples/analyze.js]: Is there a reason we need to specify the `v1` API here?  You should be able to just require without that.

File: samples/analyze.js [hunk 2]
```diff
@@ -301,6 +301,299 @@ async function analyzeVideoTranscription(gcsUri) {
   // [END video_speech_transcription_gcs]
 }
 
+async function analyzeTextGCS(gcsUri) {
```
Comment (JustinBeckwith) [inline samples/analyze.js]: Can you add a comment describing what `gcsUri` is?

File: samples/analyze.js [hunk 3]
```diff
@@ -301,6 +301,299 @@ async function analyzeVideoTranscription(gcsUri) {
...
+  const [operation] = await video.annotateVideo(request);
+  const results = await operation.promise();
+  console.log('Waiting for operation to complete...');
+  // Gets annotations for video
+  const textAnnotations = results[0].annotationResults[0].textAnnotations;
+  textAnnotations.forEach(textAnnotation => {
+    console.log(`Text ${textAnnotation.text} occurs at:`);
+    textAnnotation.segments.forEach(segment => {
+      const time = segment.segment;
+      if (time.startTimeOffset.seconds === undefined) {
```
Comment (JustinBeckwith) [inline samples/analyze.js]: This feels like a lot of boilerplate code.  Instead, in the `console.log` below, can you just do something like:

```js
console.log(`  Start: ${time.startTimeOffset.seconds || 0}.${(time.startTimeOffset.nanos / 1e6).toFixed(0)}s`);
```

File: samples/analyze.js [hunk 4]
```diff
@@ -301,6 +301,299 @@ async function analyzeVideoTranscription(gcsUri) {
...
+        time.startTimeOffset.nanos = 0;
+      }
+      if (time.endTimeOffset.seconds === undefined) {
+        time.endTimeOffset.seconds = 0;
+      }
+      if (time.endTimeOffset.nanos === undefined) {
+        time.endTimeOffset.nanos = 0;
+      }
+      console.log(
+        `\tStart: ${time.startTimeOffset.seconds}` +
```
Comment (JustinBeckwith) [inline samples/analyze.js]: Using `\t` is really hard to read.  Can we just use spaces instead?

File: samples/analyze.js (hunk 1 above)
Comment (nirupa-kumar) [reply samples/analyze.js]: This has been fixed.

File: samples/analyze.js (hunk 3 above)
Comment (nirupa-kumar) [reply samples/analyze.js]: Fixed.

File: samples/analyze.js (hunk 4 above)
Comment (nirupa-kumar) [reply samples/analyze.js]: Fixed in latest commit.

**Pull request merged**